subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1156,6 +1162,32 @@ static void le_read_buffer_size(struct net_buf *buf, struct net_buf **evt)
 	rp->le_max_num = CONFIG_BT_CTLR_TX_BUFFERS;
 }
 
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+#if defined(CONFIG_BT_CTLR_ISO_TX_BUFFERS)
+#define BT_CTLR_ISO_TX_BUFFERS CONFIG_BT_CTLR_ISO_TX_BUFFERS
+#else
+#define BT_CTLR_ISO_TX_BUFFERS 0
+#endif
+#if defined(CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE)
+#define BT_CTLR_ISO_TX_BUFFER_SIZE CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE
+#else
+#define BT_CTLR_ISO_TX_BUFFER_SIZE 0
+#endif
```

Comment:
Why is this necessary? `BT_CTLR_ISO_TX_BUFFER_SIZE` is not conditional on anything, so it's always defined.

Reply:
oh, `BT_CTLR_ISO_TX_BUFFER_SIZE` should have a `depends on`, will add it.

Reply:
I have remove the `if defined(...` checks and added the dependencies in Kconfig. 

Not going to do anything for ACL buffers wrt `CONFIG_BT_CONN` there is a different PR that removes controller Kconfig related to ACL Tx count and buffer size. 

I will rebase this PR after #34771 is merged.